kitsune/questions/static/questions/js/components/TitleContentEditor.jsx

```diff
@@ -16,6 +18,24 @@ export default class TitleContentEditor extends AAQStep {
     }
   }
 
+  setPropsFromUrl() {
+      UrlActions.getPropsFromPath("/questions/new/(.*)/(.*)", ['product', 'topic']);
+      let urlData = UrlStore.get('pathProps');
+      if (urlData.product) {
+        AAQActions.setProduct(urlData.product);
+      }
+      if (urlData.topic) {
+        AAQActions.setTopic(urlData.topic);
+      }
+  }
+
+  componentWillMount() {
+    if (!this.props.question.topic &&
+            !this.props.question.product) {
```

Comment:
Nit: In Python we have a 99 character line limit. In JS we don't have an explicit line length limit. Either way, I think this could be wrapped to one line, if you think that reads better.